### PR TITLE
[17.multilingual-bot] Add TranslationResult class

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public String getTo() {
+        return this.to;
+    }
+
+    /**
+     * Sets the target language locale.
+     * @param locale Target locale.
+     */
+    public void setTo(String locale) {
+        this.to = locale;
```
Comment (Batta32) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java:43]: why `locale`?

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.multilingual.translation.model;
+
+/**
+ * Translation result from Translator API v3.
+ */
+public class TranslationResult {
+    private String text;
+
```
Comment (Batta32) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java:11]: remove unnecesary space

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+     * Sets the translation result text.
+     * @param resultText Translation result.
+     */
+    public void setText(String resultText) {
+        this.text = resultText;
+    }
+
+    /**
+     * Gets the target language locale.
+     * @return Locale.
```
Comment (Batta32) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java:32]: why `locale`? How do you know if it's locale?

**Review by Batta32 — CHANGES_REQUESTED** [review]
Also, what about the `[JsonProperty("text")]` ([see](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/17.multilingual-bot/Translation/Model/TranslatorResult.cs)), have you found another example?

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java (hunk 3 above)
Comment (VictorGrycuk) [reply samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/translation/model/TranslationResult.java:32]: [NodeJS Sample 17](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/javascript_nodejs/17.multilingual-bot/translation/microsoftTranslator.js#L21-L23) uses the `to` param and passes the locale, so it's the target locale

**Review by Batta32 — APPROVED** [review]
LGTM

**Review by fran893 — APPROVED** [review]
LGTM

**Pull request merged**